third_party_integration/mmdetection/0001-Modifications-for-NNCF-usage.patch

```diff
@@ -92,15 +94,15 @@ index 0000000..81a90ba
 +input_size = 800
 +nncf_config = dict(compression=[dict(algorithm="quantization",
 +                                     weights=dict(per_channel=True),
-+                                     initializer=dict(range=dict(num_init_samples=10,
++                                     initializer=dict(range=dict(num_init_steps=5,
```

Comment:
Doesn't seem to be right. Have you launched the corresponding sanity test locally, at least?